src/Scene.tsx

```diff
@@ -328,35 +327,20 @@ export default function Scene(){
                  }
               }
             }
-          }
           event.target.blur()
         }
-  
         function onRedoBtn(event: any){
```

Comment:
About onRedoBtn function
1. Just in case in the top of function shuold be condition which checks length of undoArr array;
2. Instead of Element the parameter of removeCube function should be object, which should be set higher as Three js Mesh from ThreeScene.scene?.children.
There are right code for this function on the attched image
![image](https://user-images.githubusercontent.com/89405012/230680402-dc2543ea-f847-4bcd-aa3e-d6557d65611e.png)

Reply:
1. I don't think the top condition is really necessary in this case - as long as `Array.pop()` operation idempotent and returns `undefined` on an empty array, we should not really worry about the length of it
2. I would keep the `filter` in the `Controls.removeCube(object) as it would let us not repeat ourselves - we still need to find the `object` to pass it down to the function every time we use `Controls.removeCube(object)`.
Moreover, I would still pass the entire `element`, not `object` to the `Controls.removeCube(element)` function